lib/phone_number_processor.rb

```diff
@@ -6,7 +6,7 @@ class PhoneNumberProcessor
 
   def initialize
     @lookup_hash = Hash.new
-    phone_number_list = Twilio::REST::Client.new(ENV['TWILIO_SID'], ENV['TWILIO_AUTH']).account.incoming_phone_numbers.list
+    phone_number_list = Twilio::REST::Client.new('AC9059ee47e7802d8fdde65e07e0b0384a', '7493f6d49f093b405c393672f0e23fd1').account.incoming_phone_numbers.list
```

Comment:
These keys should not be hard coded and should be environment variables as before.